### ISSUE TITLE
Disabling autoplay for embedded videos does not stop autoplay.

### ISSUE THREAD

**Issue by kazy-kode** [mkhstar/suneditor-react#177]
Autoplay is enabled, even when set to false in both videoTagAttrs and videoIframeAttrs. The autoplay happens both while editing, and when the saved content created by suneditor-react is loaded in other components using dangerouslySetInnerHTML.

```js
export default function Editor(props) {
  const ref = useRef()
  const { defaultValue = '', onChange, width = '100%', showToolbar, placeholder = '' } = props

  return (
    <SunEditor
      lang="en"
      setOptions={options}
      videoTagAttrs={{
        autoplay: false,
      }}
      videoIframeAttrs={{
        autoplay: false,
      }}
      {...{ width, ref, showToolbar, placeholder, defaultValue, onChange }}
    />
  )
}

const options = {
  height: 400,
  katex,
  buttonList,
  resizingBar: false,
  showPathLabel: false,
  templates: [],
  autoplay: false,
  videoTagAttrs: {
    autoplay: false,
  },
  videoIframeAttrs: {
    autoplay: false,
  },
}

const buttonList = [
  ['bold', 'underline', 'italic', 'strike', 'subscript', 'superscript', 'formatBlock'],
  ['blockquote'],
  ['fontColor', 'hiliteColor'],
  ['removeFormat'],
  ['outdent', 'indent'],
  ['align', 'horizontalRule', 'list', 'lineHeight'],
  ['table', 'link', 'image', 'video', 'math'],
  ['template'],
]

```

This is the saved html in the database:
```html
<div class="se-component se-video-container __se__float-none" contenteditable="false" style="width: 75%">
  <figure style="height: 56.25%; padding-bottom: 56.25%; margin: 0px; width: 100%">
    <iframe
      frameborder="0"
      allowfullscreen=""
      autoplay="false"
      style="width: 100%; height: 100%"
      src="https://d12fe2gk0yrpks.cloudfront.net/134b8317-f039-4747-8374-b896271ee3d5/mp4/efec6d2b-895d-4966-a05e-19f80450b3a5_Mp4_Avc_Aac_16x9_1280x720p_24Hz_4.5Mbps_qvbr.mp4"
      data-proportion="true"
      data-percentage="75,56.25%"
      data-size="75%,56.25%"
      data-align="none"
      data-index="0"
      data-file-name="efec6d2b-895d-4966-a05e-19f80450b3a5_Mp4_Avc_Aac_16x9_1280x720p_24Hz_4.5Mbps_qvbr.mp4"
      data-file-size="0"
      data-origin="100%,56.25%"
      data-rotate=""
      data-rotatex=""
      data-rotatey=""
    ></iframe>
  </figure>
</div>
```

**Comment by lardissone** [comment]
I've read that you can't have control on the contents of an iframe, and that the easiest way to fix this is using the `<video>` tag.

If so, is there a way to use `<video>` instead of `<iframe>` for direct URL videos?

**Comment by mkhstar** [comment]
Hi @lardissone 
Thanks, fixed in 2.16.0 
autoplay: false, should disable autoplay
